### PR TITLE
Tools: fix the switch-to-uart function in TFMiniPlus script

### PR DESCRIPTION
**Pull request by tatsuy**
I changed these.
- fix the typo
- Allows the argument of the i2c address to input hexadecimal values

### CODE REVIEW

**Comment by rmackay9** [comment]
@bys1123, @lucasdemarchi, you guys are OK with this change?

**Comment by lucasdemarchi** [comment]
@tatsuy what typo are you referring to? The typo on the help message is still there.

File: Tools/scripts/tfminiplus.py [hunk 1]
```diff
@@ -46,8 +46,8 @@ def parse_args(proc_args):
     p.set_defaults(func=cmd_switch_to_i2c)
 
     p = subparsers.add_parser('switch-to-uart', help="Switch sensor to I2C mode")
```
Comment (muramura) [inline Tools/scripts/tfminiplus.py]: This is to switch from I2C to UART.

Comment (tatsuy) [reply Tools/scripts/tfminiplus.py]: Fixed. thanks!

**Comment by tatsuy** [comment]
@lucasdemarchi Fixed that the argument name of I2C_dev is different. (args.i2c_dev and 'i2c-dev')
I fixed the help messages, too.

**Comment by rmackay9** [comment]
I think I might merge this if there are no objections.  @lucasdemarchi, you're OK with this?

**Comment by lucasdemarchi** [comment]
The i2c-dev was not really a typo, but it doesn't matter much

**Comment by rmackay9** [comment]
ok merging, hope that's ok.

**Pull request merged**